packages/strapi-design-system/src/Carousel/Carousel.js

```diff
@@ -0,0 +1,121 @@
+import React, { Children, cloneElement, useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import NextFilterIcon from '@strapi/icons/NextFilter';
+import BackFilterIcon from '@strapi/icons/BackFilter';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { KeyboardKeys } from '../helpers/keyboardKeys';
+
+const CarouselGrid = styled.div`
+  display: grid;
+  grid-template-columns: auto 1fr auto;
+  grid-template-areas: 'startAction slides endAction';
+  position: relative;
```

Comment:
Can't we use the Grid component ?

Reply:
Not in this case 😊 . I wanted the focus order to remain the following: Left arow, right arrow, then actions.

Problem is that, if I wanted to keep this order, I need the two buttons to be defined in the DOM before the list of actions.

Our grid component is for a X column based grid but we don't manage areas. With the Carousel grid, I can place the element in the order I want and so position the right arrow after the slides, without breaking the tab navigation order 😊